docs/application/dotnet/tutorials/certificates/managing-certificate-profile.md

```diff
@@ -46,14 +46,14 @@ Follow the same steps to create a certificate profile as described in `4. Create
 
 3. Create new Distributor for certificate profile
 
-    Follow the same steps as mentioned in `1. Create new or select existing distributor certiicate` of [Create a New Distributor Certificate](creating-certificates.md#create-a-new-distributor-certificate).
+    Follow the same steps as mentioned in [1. Create new or select existing distributor certiicate](creating-certificates.md#create_new_or_select_existing_distributor_certificate).
 
 ## Importing Existing Certificates
 
 If you already have both the **author.p12** and **distributor.p12** certificates and you do not require a new certificate from Samsung, you can import and use the available certificates to sign your application.
 In that case, create a new certificate profile and select existing certificates at each step.
 
-Follow the same steps as mentioned in creating certificate before the step `4. Create new or select old profile` of [Creating Certificate Profile](creating-certificates.md#creating-certificate-profile)
+Follow the same steps as mentioned in creating certificate before the step [4. Create new or select old profile](creating-certificates.md#create_new_or_select_old_profile)
```

Comment:
Follow the same steps as mentioned in creating certificate before [creating a new profile or selecting an old profile](creating-certificates.md#create_new_or_select_old_profile).

Reply:
done

Reply:
@Reni-90 In this sentence, the information which should be focused is "creating certificate", not "creating a new profile or selecting an old profile". So how about linking to "#creating-certificate-profile" and change the sentence to make user focus on "creating certificate", too?

Reply:
Yes I completely agree with you.

Can we rewrite the sentence as:

Follow the same steps mentioned in [creating certificates](creating-certificates.md#create_new_or_select_old_profile) before creating a new profile or selecting an old profile.

Reply:
@Reni-90 , would you please check your comment again?
You are still asking to link to "#create_new_or_select_old_profile", which is "not the information to be focused".

Reply:
Sorry.
Can we rewrite the sentence as:

Follow the same steps mentioned in [creating certificates](creating-certificates.md#creating-certificate-profile) before creating a new profile or selecting an old profile.